documentation/loginAndLogoutFlow.md

```diff
@@ -0,0 +1,51 @@
+## User login and Logout flow
+
+This document describes the flow on a locally installed GPII system when the user logs in and logs out. First an overview of the endpoints and related semantics are given and this will be followed by a more in-depth technical walkthrough of the flow.
+
+### Overview and APIs
+We support 3 different logon related URLs, namely:
+* `/user/:token/logonChange` - change the logon state of the given token (ie. log in or out)
+ ** If no user is logged in, user <mytoken> will be logged in
+ ** If <mytoken> is logged in, <mytoken> will be logged out
+ ** If another user is logged in, nothing happens
```

Comment:
Should mention that "nothing happens and an error response will be returned", also below